powerbi-docs/developer/office-store.md

```diff
@@ -36,7 +36,7 @@ One you're done coding and testing your custom visual and have packaged it into
 | --- | --- | --- |
 | Pbiviz package contains all the required metadata |Yes |Visual name<br>Display name<br>GUID<br>Version<br>Description<br>Author name and email |
 | Sample .pbix report file |Yes |To showcase your visual, you should help users to get acquainted with the visual. You should emphasize on the value the visual brings to the user and give examples of usage, formatting options, etc. You can also add a *"hints"* page at the end with some tips and tricks, things to avoid and the like.<br>The sample .pbix report file must work offline, without any external connection |
-| Icon |Yes |You should include the custom visual logo that will appear in the store front. The format can be .png, .jpg, .jpeg or .gif. It must be exactly 300px (width) x 300px (height). It can be no larger than 512kb for the file size. |
+| Icon |Yes |You should include the custom visual logo that will appear in the store front. The format can be .png, .jpg, .jpeg or .gif. It must be exactly 300px (width) x 300px (height). It can be no larger than 512kb for the file size. Please review the short guide carefully before submitting the Icon https://docs.microsoft.com/en-us/office/dev/store/craft-effective-appsource-store-images |
```

Comment:
I would change it to this - You should review the short guide carefully before submitting your logo [here](https://docs.microsoft.com/en-us/office/dev/store/craft-effective-appsource-store-images).